sdk/include/xdk/ntddk.template.h

```diff
@@ -75,9 +75,7 @@ struct _DISK_GEOMETRY_EX;
 /* Structures not exposed to drivers */
 typedef struct _BUS_HANDLER *PBUS_HANDLER;
 typedef struct _DEVICE_HANDLER_OBJECT *PDEVICE_HANDLER_OBJECT;
-#if defined(_NTHAL_INCLUDED_)
```

Comment:
Why?

Reply:
I did not see it being defined anywhere and it was required for `HalReportIdleStateUsage` definition, so I dropped it.

Reply:
True, indeed, although it's present in the official ntddk for this typedef and the KPROCESS, KTHREAD ones too.
May I then suggest
```suggestion
#if defined(_NTHAL_INCLUDED_) || defined(__REACTOS__)
```
:thinking: